articles/puppet-module.md

```diff
@@ -20,9 +20,9 @@ Install [Fleet's Puppet module](https://forge.puppet.com/modules/fleetdm/fleetdm
 
 ### Step 2: configure Puppet to talk to Fleet using Heira
 
-1. In Fleet, create an API-only user with the GitOps role. Instructions for creating an API-only user are [here](./fleetctl-CLI.md#create-an-api-only-user).
+1. In Fleet, create an API-only user with the GitOps role. Instructions for creating an API-only user are [here](https://fleetdm.com/guides/fleetctl#create-api-only-user).
 
-2. Get the API token for your new API-only user. Learn how [here](./fleetctl-CLI.md#get-the-api-token-of-an-api-only-user).
+2. Get the API token for your new API-only user. Learn how [here](https://fleetdm.com/docs/rest-api/rest-api#log-in).
```

Comment:
This heading doesn't seem to exist in any guides. The closest I could find for the topic was in the REST API docs.

Reply:
Is it better to link to https://fleetdm.com/docs/rest-api/rest-api#retrieve-your-api-token instead?

Reply:
@gillespi314 As of 4.53.0, the API token is automatically shown in the CLI after creating an API-only user, so I think we can just mention that here instead of linking out anywhere. (See https://github.com/fleetdm/fleet/issues/16961)